src/main/java/com/algorand/algosdk/v2/client/model/TransactionSignature.java

```diff
@@ -0,0 +1,101 @@
+package com.algorand.algosdk.v2.client.model;
+
+import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+
+/**
+ * Validation signature associated with some data. Only one of the signatures 
+ * should be provided. 
+ */
+public class TransactionSignature {
```

Comment:
Could these use our existing signature types? I think they should match exactly.